components/src/molecules/cookie-message/CookieMessage.html

```diff
@@ -0,0 +1,30 @@
+<div
+  :class="['a-cookie-message', isOpen && 'a-cookie-message--open']"
+  :data-type="dataType"
+  :aria-label="ariaLabel"
+  tabindex="0"
+>
+  <div class="a-cookie-message__container">
+    <div class="a-cookie-message__text">
+      <!-- @slot Content slot -->
+      <slot />
+    </div>
+    <!-- @slot Close button (Scoped slot) -->
+    <slot
+      name="button"
+      :close-bar="closeBar"
+    >
+      <a-button
+        :aria-label="closeAriaLabel"
+        class="a-cookie-message__close"
+        @click.native="closeBar()"
+      >
+        {{ closeButton }}
+        <a-icon
+          :icon="closeIcon"
```

Comment:
I think you can change it to `icon="close"` and remove `closeIcon` prop to reduce the numer of props (slot can be used to override the icon)